### ISSUE TITLE
The loss value of the pytorch version of the code fluctuates, and the effect is not as good as tf

### ISSUE THREAD

**Issue by HKLFJS** [xavysp/DexiNed#59]
Hello author, why is the effect of the model trained by tf better than pytorch? ? Another problem is that when we use pytorch for training, the loss value is difficult to drop, and the loss appears to oscillate in the later stage of the training, but the output effect of the network is OK. May I ask the author if he also encountered it during training The same situation? ? ? Looking forward to your reply

**Comment by xavysp** [comment]
Hi @HKLFJS,

I am afraid I cannot give you a consistent answer because, I saw that difference more than a year ago  but I cannot find an answer. However, In the coming days I will be updating the DexiNed-Pytorch and the new version of BIPED will be released, hope it will give us a better understanding of this situation.

Cheers,
P.S. the loss values depends on tf.nn.weighted_cross_entropy_with_logits in TF and F.binary_cross_entropy_with_logits in Pytorch, we should check such implementations.

**Comment by lyfv1** [comment]
Could you send the pytorch-code version to me(there is not the version in this page), thanks a lot.

**Comment by xavysp** [comment]
Sorry, I did not give you the requirement in the right time
